usr/src/cmd/ypcmd/yppasswd/yplckpwdf.c

```diff
@@ -67,7 +67,7 @@ yplckpwdf()
 	flock.l_type = F_WRLCK;
 	(void) sigset(SIGALRM, almhdlr);
 	(void) alarm(S_WAITTIME);
-	retval = fcntl(fildes, F_SETLKW, (int)&flock);
+	retval = fcntl(fildes, F_SETLKW, (intptr_t)&flock);
```

Comment:
I would just remove the cast entirely here (and below). yp is the only place under `usr/src/cmd` that had it.